Library/Formula/v8.rb

```diff
@@ -20,6 +22,8 @@ class V8 < Formula
   depends_on :macos => :lion
 
   depends_on :python => :build # gyp doesn't run under 2.6 or lower
+  depends_on "pkg-config" => :build
+  depends_on "icu4c"
```

Comment:
This should probably be `optional`. It's kind of a heavy dep to enforce on people, so a bit reluctant to do this unconditionally given I think you may be the first person who has asked for ICU support on the formula, heh.